jfr-daemon/src/main/java/com/newrelic/jfr/daemon/SetupUtils.java

```diff
@@ -88,6 +87,23 @@ public static JFRUploader buildUploader(DaemonConfig config) {
     return new JFRUploader(new NewRelicTelemetrySender(telemetryClient), recordedEventBuffer);
   }
 
+  /**
+   * Build a {@link JfrController} with the {@code config} and {@code uploader}.
+   *
+   * <p>This method is called by the New Relic Java Agent.
+   *
+   * @param config the config
+   * @param uploader the uploader
+   * @return the JfrController
+   */
+  public static JfrController buildJfrController(DaemonConfig config, JFRUploader uploader) {
+    FileJfrRecorderFactory recorderFactory =
+        new FileJfrRecorderFactory(config.getHarvestInterval());
+    final JfrController jfrController =
+        new JfrController(recorderFactory, uploader, config.getHarvestInterval());
+    return jfrController;
```

Comment:
This could be simplified to just:

```
    return new JfrController(recorderFactory, uploader, config.getHarvestInterval());
```

Since the `jfrController` variable doesn't serve a purpose.